app/controllers/shared/api.php

```diff
@@ -362,30 +363,33 @@
         if ($useCache) {
             $resource = null;
             $data = $response->getPayload();
+
             if (!empty($data['payload'])) {
                 $pattern = $route->getLabel('cache.resource', null);
                 if (!empty($pattern)) {
                     $resource = $parseLabel($pattern, $responsePayload, $requestParams, $user);
                 }
 
                 $key = md5($request->getURI() . implode('*', $request->getParams()));
-
                 $data = json_encode([
                 'content-type' => $response->getContentType(),
                 'payload' => base64_encode($data['payload']),
                 ]) ;
 
                 $signature = md5($data);
                 $cacheLog  = $dbForProject->getDocument('cache', $key);
+                $accessedAt = $cacheLog->getAttribute('accessedAt', '');
+                $accessedAt = (new \DateTime($accessedAt))->format('Y/m/d');
+
                 if ($cacheLog->isEmpty()) {
                     Authorization::skip(fn () => $dbForProject->createDocument('cache', new Document([
                     '$id' => $key,
                     'resource' => $resource,
-                    'accessedAt' => \time(),
+                    'accessedAt' => DateTime::now(),
```

Comment:
```suggestion
                    'accessedAt' => \DateTime(),
```


`DateTime::now()` returns a formatted string. We need to use the DateTime object directly as it then goes through the database filters